config.yml.example

```diff
@@ -6,7 +6,7 @@ user: git
 # You only have to change the default if you have configured Unicorn
```

Comment:
Maybe it would make sense to have a simple config.yml baked into the image that doesn't depend on env vars, and then overwrite it with the ConfigMap?